addons/website_sale/tests/__init__.py

```diff
@@ -11,5 +11,6 @@
 from . import test_website_sale_product_attribute_value_config
 from . import test_website_sale_image
 from . import test_website_sequence
+from . import test_website_sale_shop
```

Comment:
Filename is a bit too generic, it would be better if the name was related to the content of the tests.
Especially since the test is so specific.